lib/schema-converters.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const { iter } = require('metarhia-common');
+const { all: decorators } = require('./decorators');
+
+const defaultOptions = {
+  useEsm: false,
+};
+
+const createImport = name => `import { ${name} } from 'metaschema';`;
+const createRequire = name => `const { ${name} } = require('metaschema');`;
+
+const generateImports = (usedDecorators, useEsm) =>
+  // eslint-disable-next-line implicit-arrow-linebreak
+  `${(useEsm ? createImport : createRequire)('decorators')}\n` +
+  `const { ${usedDecorators.join(', ')} } = decorators;`;
+
+const toJs = (source, options = {}) => {
+  const { useEsm } = Object.assign({}, defaultOptions, options);
+  const usedDecorators = iter(Object.keys(decorators))
+    .filter(name => source.match(`${name}\\([\\S\\s]*\\)`))
```

Comment:
This doesn't account for possible whitespace between the identifier and the left paren, and for possible identifier part to the left of `${name}`. I'd just match for `\\b${name}\\b`, but if you think that checking for parens is absolutely necessary, it should be something like `\\b${name}\\s*(`.